src/ui/zcl_abapgit_gui_chunk_lib.clas.abap

```diff
@@ -778,4 +788,31 @@ CLASS ZCL_ABAPGIT_GUI_CHUNK_LIB IMPLEMENTATION.
     ro_html->add( '</div>' ).
 
   ENDMETHOD.
+  METHOD render_repo_top_commit_hash.
+
+    DATA: lv_commit_hash       TYPE zif_abapgit_definitions=>ty_sha1,
+          lv_commit_short_hash TYPE zif_abapgit_definitions=>ty_sha1,
+          lv_repo_url          TYPE zif_abapgit_persistence=>ty_repo-url,
+          lv_display_url       TYPE zif_abapgit_persistence=>ty_repo-url,
+          lv_icon_commit       TYPE string.
+
+    lv_commit_hash = iv_repo_online->get_sha1_remote( ).
+    lv_commit_short_hash = lv_commit_hash(7).
+
+    TRY.
+        lv_display_url = iv_repo_online->get_commit_display_url( lv_commit_hash ).
+
+        lv_icon_commit = zcl_abapgit_html=>icon( iv_name  = 'code-commit'
+                                                 iv_class = 'pad-sides'
+                                                 iv_hint  = 'Commit' ).
+        iv_html->add_a( iv_txt   = |{ lv_icon_commit }{ lv_commit_short_hash }|
+                        iv_act   = |{ zif_abapgit_definitions=>c_action-url }?|
+                                && lv_display_url
+                        iv_class = |url| ).
+      CATCH zcx_abapgit_exception.
+        iv_html->add( |<span class="url">{ lv_commit_short_hash }</span>|  ).
```

Comment:
Ah, no. One minor bug. If the "provider not supported" - e.g. gitlab - then icon is not rendered
![image](https://user-images.githubusercontent.com/15635498/90066277-30fa5880-dcf6-11ea-881d-c8e46d5f327a.png)

BTW: gitlab commit urls are: `https://gitlab.com/<user>/<repo>/-/commit/<hash>` 😉 🙏

Reply:
only online repo can have a "show url" and should know its provider and show url...

gitlab added - further provider => #3733